specifyweb/frontend/js_src/lib/components/WorkBench/CellMeta.ts

```diff
@@ -347,6 +368,15 @@ export class WbCellMeta {
           count + (this.getCellMetaFromArray(info, 'isModified') ? 1 : 0),
         0
       ),
+      updatedCells: cellMeta.reduce(
+        (count, info)=>count + (this.getCellMetaFromArray(info, 'isUpdated') ? 1 : 0), 0
+      ),
+      deletedCells: cellMeta.reduce(
+        (count, info)=>count + (this.getCellMetaFromArray(info, 'isDeleted') ? 1 : 0), 0
+      ),
+      matchedAndChangedCells: cellMeta.reduce(
+        (count, info)=>count + (this.getCellMetaFromArray(info, 'isMatchedAndChanged') ? 1 : 0), 0
+      )
```

Comment:
(optional) a small performance optimization - could you short circuit the computation of all of these if you are not in an batch editor?

also, we could make this a bit less "functional" but changing code like this:
```ts
let modifiedCells = 0;
let ... = 0;
...

for(const meta of cellMeta) {
  if(this.getCellMetaFromArray(info, 'isDeleted')) modifiedCells += 1;
  /// ...
  if(isInBatchEditor) {
    if(...) updatedCells += 1;
  }
}
``